src/Plugin/CaptureUtility/ScreenshotCaptureUtility.php

```diff
@@ -39,4 +55,29 @@ public function getResponse() {
     return $this->response;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function missingDependencies() {
+    $capture_message = $this->t('Screen Capture package missing.');
```

Comment:
Eh, the message format in the lines below make it funny with the class names... nbd